client/sites/hosting/components/hosting-callout/index.tsx

```diff
@@ -15,13 +15,20 @@ export function HostingActivationCallout( {
 	siteId: number;
 	redirectUrl?: string;
 } ) {
-	const { data: siteTransferData } = useSiteTransferStatusQuery( siteId, {
-		refetchIntervalInBackground: true,
-	} );
+	const { data: siteTransferData, refetch: fetchSiteTransferStatus } = useSiteTransferStatusQuery(
+		siteId,
+		{
+			refetchIntervalInBackground: true,
+		}
+	);
 
 	const isActivating = siteTransferData?.isTransferring;
 	const isActivated = transferStates.COMPLETED;
 
+	useEffect( () => {
+		fetchSiteTransferStatus();
+	}, [ fetchSiteTransferStatus ] );
+
 	useEffect( () => {
 		if ( isActivated && redirectUrl ) {
```

Comment:
We don't get a `redirectUrl` passed to this component. Can this be why the callout is stuck? 🤔 

<img width="3200" height="2650" alt="Image" src="https://github.com/user-attachments/assets/057ca646-7790-43c8-b3ff-fb9919741b7a" />

Alternatively, I think this might get stuck if we're not getting the site data refetched in the [parent component](https://github.com/Automattic/wp-calypso/blob/ff3dcadbe3cded3c9090f267fa236611ab5f431d/client/sites/hosting/controller.tsx#L20). Here, the site data is stored/refetched via Redux.